src/ascii_dialog/guess.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python3
+
+from sasdata.dataset_types import DatasetType
```

Comment:
Does this logic belong in `sasdata`? That is, could it be used by a script, a jupyter notebook or a web gui?

Reply:
It will be used by a file loader in sasdata too.